dashboard/src/components/ErrorAlert/ErrorAlert.tsx

```diff
@@ -0,0 +1,43 @@
+import Alert from "components/js/Alert";
+import RpcErrorMessage from "components/RpcErrorMessage";
+import { RpcError } from "shared/RpcError";
+import { CustomError } from "shared/types";
+import "./ErrorAlert.css";
+
+export interface IErrorAlert {
+  children: CustomError | Error | string;
```

Comment:
Component supports rendering this three types.

Reply:
What if there are no children?

Reply:
Children is a mandatory item it seems. In this case doesn't make sense to show the error block without any content, does it?